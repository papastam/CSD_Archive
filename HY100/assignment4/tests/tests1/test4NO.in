4
16 3 2 13
5 10 11 5
9 6 7 12
4 15 14 1
