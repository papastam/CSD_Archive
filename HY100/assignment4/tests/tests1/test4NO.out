ROW 1 YES
ROW 2 NO
ROW 3 YES
ROW 4 YES
COLUMN 1 YES
COLUMN 2 YES
COLUMN 3 YES
COLUMN 4 NO
DIAG1 YES
DIAG2 YES
UNIQUE NO
MAGIC NO
